internal/plugins/linux/rpm.go

```diff
@@ -136,11 +136,9 @@ func (p *rpmPlugin) Run() {
 		return
 	}
 
-	err = watcher.Add("/var/lib/rpm/.rpm.lock")
+	// Listening writes to a RPM database file used on packages modifications
+	err = watcher.Add("/var/lib/rpm/Installtid")
 	if err != nil {
-		// Some old distros, like SLES 11, do not provide .rpm.lock file, but the same
-		// effect can be achieved by listening some standard files from the RPM database
-		err = watcher.Add("/var/lib/rpm/Installtid")
 		if err != nil {
```

Comment:
one `if err != nil {` can be removed now 😉

Reply:
Totally right! Thanks